superduperdb/misc/runnable/thread.py

```diff
@@ -1,12 +1,29 @@
 import dataclasses as dc
-import sys
+import traceback
 import typing as t
-from functools import cached_property, partial
+from functools import cached_property
 from threading import Thread
 
+import dek
```

Comment:
haven't seen his before ?

Im not sure if we should add deps for a decorator 
may be the team can decide it better

Reply:
We already have this dependency, transitively!

I'll rewrite it as a classic decorator and you can see the difference, make up your mind. :-)